AdvanceTaskMarsPart2/Utilities/GlobalHelpers.cs

```diff
@@ -0,0 +1,123 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AdvanceTaskMarsPart2.Model;
+using Newtonsoft.Json;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using OpenQA.Selenium.Support.UI;
+
+namespace AdvanceTaskMarsPart2.Utilities
+{
+    public class GlobalHelpers
+    {
+        
+        public static IWebDriver driver;
+        public static ExtentReports extent;
+        public static ExtentTest test;
+
+        //File path where report is generated
+        private static string reportPath = "extentReports.html";
+        public void Initialise()
+        {
+            driver = new ChromeDriver();
```

Comment:
for future reference you can move the driver to OneTimeSetUp as it needs to be only setup once, over all your task looks good